src/util/kaldi-table-inl.h

```diff
@@ -340,6 +410,12 @@ template<class Holder>  class SequentialTableReaderScriptImpl:
     // or the user called FreeCurrent().. note,
     // if when called by user we are in this state,
     // it means the user called FreeCurrent().
+    kObjectLoaded, // Called NextScpLine() and found the next    yes         yes
+    // object we need was alreay loaded in the last line.
```

Comment:
Yes I think so.
Anyway make sure to describe it accurately.

On Thu, Mar 24, 2016 at 12:38 AM, Xiaohui Zhang <notifications@github.com>
wrote:

> In src/util/kaldi-table-inl.h
> <https://github.com/kaldi-asr/kaldi/pull/611#discussion_r57273975>:
>
> > @@ -340,6 +410,12 @@ template<class Holder>  class SequentialTableReaderScriptImpl:
> >      // or the user called FreeCurrent().. note,
> >      // if when called by user we are in this state,
> >      // it means the user called FreeCurrent().
> > +    kObjectLoaded, // Called NextScpLine() and found the next    yes         yes
> > +    // object we need was alreay loaded in the last line.
>
> Did you mean LoadCurrent()? there is no LoadObject() here. Yep "last" is
> not accurately. I think in my implementation kObjectLoaded just describes:
> we already have an object in the holder, corresponding to data_rxfilename_
> in the *next* line. It doesn't matter that the range was specified or not
> (in the current line or next line)
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/611/files/8d76a41bf70954b70888f9188d737a1beb7c66b2#r57273975>
>